internal/rbd/rbd_attach.go

```diff
@@ -259,6 +269,10 @@ func appendNbdDeviceTypeAndOptions(cmdArgs []string, isThick bool, userOptions s
 			cmdArgs = append(cmdArgs, "--options", fmt.Sprintf("%s=%d", setNbdIOTimeout, defaultNbdIOTimeout))
 		}
 
+		if cookie != "" && util.IsVersionSatisfied(rbdNbdCookieSupportVersion, rbdNbdVersion) {
```

Comment:
move this to a helper function, like `cmdArgs = appendNbdCookieOption(cmdArgs, cookie)` or such

Reply:
Ok

Reply:
after the recent changes, looks like this is not needed for me.